astropy/wcs/src/wcslib_wrap.c

```diff
@@ -4014,13 +4018,34 @@ PyWcsprm_get_aux(
   return result;
 }
 
+
+static PyObject*
+PyWcsprm_get_cel(
+    PyWcsprm* self,
+    /*@unused@*/ void* closure) {
+
+  PyObject* result;
+
+    // If wcsprm.cel is not initialized, we should do so here so that users can
+    // set celestial parameters on an empty WCS.
+
+    //if (self->x.cel == 0x0) {
+      //celini(1, &self->x);
+    //}
```

Comment:
Is there still a decision to be made here?

Reply:
No. `x.cel` always exists. It is not a pointer.

Reply:
No. `x.cel` always exists. It is not a pointer.